### DISCUSSION TITLE
My breakpoint can't work.

### DISCUSSION THREAD

**Discussion by Hao-yiwen** [react-native-community/discussions-and-proposals#754] [Feedback]
I use 0.73.2 react-native,and enigne is hermes.But I can't debugger with flipper,chrome inspect and experimental debugger tools.What should I do when I want to debugger in react-natvie.

<img width="682" alt="image" src="https://github.com/react-native-community/discussions-and-proposals/assets/42827938/03f2f3c9-2b53-45e0-898a-c189a38709cf">

**Comment by kelset** [comment]
👋 there - as stated in the README and the issue template, this repo is for long-form discussions. You should be posting this issue on the main [react native repo](https://github.com/facebook/react-native).

**Reply by Hao-yiwen** [reply]
okk.
